src/renderer/components/kube-object/kube-object-meta.tsx

```diff
@@ -22,10 +23,21 @@ export class KubeObjectMeta extends React.Component<KubeObjectMetaProps> {
     return hideFields.includes(field);
   }
 
+  renderObjectName() {
+    const { object } = this.props;
+    const statuses = kubeObjectStatusRegistry.getItemsForObject(object);
+
+    if (statuses.length === 0) {
+      return object.getName();
+    }
+
+    return <>{object.getName()} <KubeObjectStatusIcon key="icon" statuses={statuses} /></>;
```

Comment:
To keep single source of truth rule, we need to allow to pass only `statuses` prop or `object`, not both. I see that `statuses` was introduces for this specific case in `kube-object-meta.tsx`?

Reply:
This will allow also to remove prop type dualism here 
```
export class KubeObjectStatusIcon extends React.Component<StatusProps | ObjectProps> {
```